examples/ssl_exporter.yaml

```diff
@@ -5,6 +5,7 @@ modules:
     prober: https
     tls_config:
       insecure_skip_verify: true
+      renegotiation: freely
```

Comment:
Could you create a dedicated example for changing renegotiation? I wouldn't want to imply that renegotiation is required for making an 'insecure' connection.

Reply:
Will do.